tests/aggregation/tests.py

```diff
@@ -1750,6 +1750,15 @@ def __init__(self, expr):
             ],
         )
 
+    def test_order_by_aggregate_default_alias(self):
+        publisher_books = (
+            Publisher.objects.values("book")
+            .annotate(Count("book"))
+            .order_by("book__count", "id")
+            .values_list("book", flat=True)
+        )
+        self.assertQuerySetEqual(publisher_books, [None, 1, 3, 2, 4, 5, 6])
```

Comment:
```suggestion
        self.assertSequenceEqual(publisher_books, [None, 1, 3, 2, 4, 5, 6])
```

Reply:
May I ask why we'd prefer `assertSequenceEqual` over `assertQuerySetEqual` here? It looks like `assertQuerySetEqual` is used just above with a `list[dict]` with `values`.